src/dtgtk/paint.h

```diff
@@ -270,6 +270,9 @@ void dtgtk_cairo_paint_display_wavelet_scale(cairo_t *cr, gint x, gint y, gint w
 /** Paint a auto level icon for retouch*/
 void dtgtk_cairo_paint_auto_levels(cairo_t *cr, gint x, gint y, gint w, gint h, gint flags, void *data);
 
+/** Paint a link icon for basic adjustments*/
```

Comment:
In order to be picked up by `show_icons_in_paint_c.pl` tool the comment has to end with space before `*/`
```suggestion
/** Paint a link icon for basic adjustments */
```